custom/icds_reports/static/js/directives/location-filter/location-filter.directive.js

```diff
@@ -433,6 +433,15 @@ function LocationFilterController($rootScope, $scope, $location, $uibModal, loca
         }
     };
 
+    // helpers for mobile dashboard - mainly pass-throughs to locations service
```

Comment:
these functions are defined on both `LocationModalController` (which mobile doesn't use) and `LocationFilterController` (which mobile does use). It's quite confusing how each is used on the web side, and hopefully a bit more clear on the mobile side. Anyway, that's why they were extracted to the service and added here.